crates/control/src/motion/look_at.rs

```diff
@@ -89,11 +91,11 @@ impl LookAt {
         }
 
         let (target, camera) = match head_motion {
-            HeadMotion::LookAt { target, camera } => (*target, *camera),
+            HeadMotion::LookAt { target, camera } => (target, camera),
             HeadMotion::LookLeftAndRightOf { target } => {
                 let left_right_shift = vector![
                     0.0,
-                    f32::tan(*context.glance_angle) * distance(target, &Point2::origin())
+                    f32::tan(*context.glance_angle) * distance(target, Point2::origin())
```

Comment:
same as `.norm()`?

Reply:
It is, yes. Do you find `target.coords().norm()` more readable than `distance(target, Point2::origin())`?

Reply:
lets keep it `distance`